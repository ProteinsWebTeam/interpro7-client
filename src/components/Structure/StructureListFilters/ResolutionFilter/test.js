import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { ResolutionFilter } from '.';

const renderer = new ShallowRenderer();

describe('<ResolutionFilter />', () => {
  test('should render', () => {
    renderer.render(
      <ResolutionFilter
        customLocation={{
          description: {
            main: {
              key: 'structure',
            },
            structure: {
              db: 'PDB',
            },
          },
          search: {},
        }}
        goToCustomLocation={() => {}}
      />,
    );
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
