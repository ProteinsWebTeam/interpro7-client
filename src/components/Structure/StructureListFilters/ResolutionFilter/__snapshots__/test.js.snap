// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResolutionFilter /> should render 1`] = `
<div
  className="column"
>
  <label
    className="radio-btn-label checked"
  >
    <input
      checked={true}
      className="radio-btn"
      disabled={false}
      name="resolution"
      onChange={[Function]}
      type="radio"
      value="All"
    />
    <span>
      All
    </span>
  </label>
  <label
    className="radio-btn-label"
  >
    <input
      checked={false}
      className="radio-btn"
      disabled={false}
      name="resolution"
      onChange={[Function]}
      type="radio"
      value="Subset"
    />
    <span>
      0
       - 
      20
       Ã…
    </span>
  </label>
  <MultipleInput
    aria-label="resolution range"
    className="range hideable hidden"
    max={20}
    maxValue={20}
    min={0}
    minValue={0}
    onChange={[Function]}
    step="0.05"
  />
</div>
`;
